components/table/style/patch.less

```diff
@@ -140,12 +140,7 @@ nz-table {
 
 .@{table-prefix-cls}-small {
   &.@{table-prefix-cls}-bordered {
-    .@{table-prefix-cls}-thead > tr > th:last-child,
```

Comment:
do not remove these lines
https://github.com/NG-ZORRO/ng-zorro-antd/issues/3373